src/frontend/src/pages/Index/Settings/AccountSettings/SecurityContent.tsx

```diff
@@ -329,3 +336,85 @@ function MfaContent() {
     </>
   );
 }
+
+function TokenContent() {
+  const { isLoading, data, refetch } = useQuery({
+    queryKey: ['token-list'],
+    queryFn: () =>
+      api.get(apiUrl(ApiEndpoints.user_tokens)).then((res) => res.data)
+  });
+
+  function revokeToken(id: string) {
+    api
+      .delete(apiUrl(ApiEndpoints.user_tokens, id))
+      .then(() => {
+        refetch();
+      })
+      .catch((res) => console.log(res.data));
+  }
+  const rows = useMemo(() => {
+    if (isLoading || data === undefined) return null;
+    return data.map((token: any) => (
+      <Table.Tr key={token.id}>
+        <Table.Td>
+          <YesNoButton value={token.active} />
+        </Table.Td>
+        <Table.Td>{token.expiry}</Table.Td>
+        <Table.Td>{token.last_seen}</Table.Td>
+        <Table.Td>{token.token}</Table.Td>
+        <Table.Td>{token.name}</Table.Td>
+        <Table.Td>
+          {token.in_use ? (
+            <Trans>Token is used - no actions</Trans>
+          ) : (
+            <Button
+              onClick={() => revokeToken(token.id)}
+              color="red"
+              disabled={!token.active}
+            >
+              <Trans>Revoke</Trans>
+            </Button>
+          )}
+        </Table.Td>
+      </Table.Tr>
+    ));
+  }, [data, isLoading]);
+
+  /* renderer */
+  if (isLoading) return <Loader />;
+
+  if (data.length == 0)
+    return (
+      <Alert icon={<IconAlertCircle size="1rem" />} color="green">
+        <Trans>No tokens configured</Trans>
+      </Alert>
+    );
+
+  return (
+    <Table stickyHeader striped highlightOnHover withTableBorder>
```

Comment:
Q: is there a reason why you built this table yourself and didn't used the inventree table, to reduce duplicate code?

Reply:
It took up a lot of unnecessary horizontal space and the complexity is not needed here

Reply:
ok